### PR TITLE
Implement edit content integration test

### PR DESCRIPTION
**Pull request by ruslanmalogulko**
- [x] Go to "content" mode
- [x] Test double-click to edit on a plugin
- [x] Make a change
- [x] Check that the change is applied
- [x] Check that double click to edit works on structure mode
- [x] Check that the edit icon works on structure mode

### CODE REVIEW

File: cms/tests/frontend/integration/editContent.js [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+        // check if content updated
+        .waitWhileVisible('.cms-modal-open', function () {
+            // ensure content updated with new one
+            test.assertSelectorHasText(
+                '.cms-plugin:last-child',
+                randomText,
+                messages.contentUpdatedDoubleClick
+            );
+            // ensure we're getting rid of previous content
+            test.assertSelectorDoesntHaveText(
```
Comment (dhorelik) [inline cms/tests/frontend/integration/editContent.js]: do we really have to check that `previousContentText` isn't there? if `.cms-plugin:last-child` content is new, it makes sense to suggest that it isn't old =)

Comment (dhorelik) [inline cms/tests/frontend/integration/editContent.js]: this would also make getting `previousContent` above unnecessary

Comment (ruslanmalogulko) [inline cms/tests/frontend/integration/editContent.js]: you're right, it's not obligatory to check previous content.
as about `previousContent` variable - it's used next to test further changes (preventing from creating new random string)

**Comment by dhorelik** [comment]
:+1:

**Pull request merged**